.github/workflows/test.yml

```
@@ -28,21 +29,72 @@ jobs:
       - name: Install dependencies
         shell: bash -l {0}
         run: |
-          conda install python=3.12 pytest coverage numpy numba conda-build
+          conda install python=3.12 pytest coverage numpy numba conda-build clangxx=14 numba/label/dev::llvmlite setuptools_scm cython pytest-cov
       - name : Display packages and system info
         shell: bash -l {0}
         run: |
           python -m numba -s
       - name: Install numba-pixie with pip
         shell: bash -l {0}
         run: |
-          python -m pip install -vv -e .
+          python -m pip install -e . --no-deps
```

```suggestion
          python -m pip install --no-deps --no-index --no-build-isolation -vv -e .
```
perhaps do this to use what is already present in the env and prevent any attempt at downloading dependencies?